python/tests/integration/arcticdb/version_store/test_basic_version_store.py

```diff
@@ -215,6 +202,66 @@ def test_unhandled_chars_already_present_update(object_version_store, unhandled_
     assert len(vitem.data) == len(new_vitem.data)
 
 
+# See AN-765 for why we need no_symbol_list fixture
+def test_large_symbols(basic_store_no_symbol_list):
+    # The following restrictions should be checked in the cpp layer's name_validation
+    MAX_SYMBOL_SIZE = 254
```

Comment:
Why has this changed from 255?

Reply:
This is the actual max symbol size:
https://github.com/man-group/ArcticDB/blob/master/cpp/arcticdb/entity/serialized_key.hpp#L87-L93
The old implementation of this test had to use `MAX_SYMBOL_SIZE - 1` as the max size to work